<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionalAudioViewer</class>
 <widget class="QDialog" name="PositionalAudioViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Positional Audio Viewer</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="1" column="2">
    <widget class="QGroupBox" name="cameraDirGroup">
     <property name="title">
      <string>Camera direction</string>
     </property>
     <layout class="QFormLayout" name="cameraDirLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cameraDirLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>cameraDirX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="cameraDirX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cameraDirLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>cameraDirY</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cameraDirY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cameraDirLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>cameraDirZ</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cameraDirZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="playerPosGroup">
     <property name="title">
      <string>Player position</string>
     </property>
     <layout class="QFormLayout" name="playerPosLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="playerPosX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="playerPosLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>playerPosX</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="playerPosY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="playerPosLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>playerPosY</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="playerPosZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="playerPosLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>playerPosZ</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="playerAxisGroup">
     <property name="title">
      <string>Player axis</string>
     </property>
     <layout class="QFormLayout" name="playerAxisLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="playerAxisLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>playerAxisX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="playerAxisX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="playerAxisLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>playerAxisY</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="playerAxisY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="playerAxisLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>playerAxisZ</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="playerAxisZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="playerDirGroup">
     <property name="title">
      <string>Player direction</string>
     </property>
     <layout class="QFormLayout" name="playerDirLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="playerDirLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>playerDirX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="playerDirX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="playerDirLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>playerDirY</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="playerDirY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="playerDirLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>playerDirZ</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="playerDirZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="cameraAxisGroup">
     <property name="title">
      <string>Camera axis</string>
     </property>
     <layout class="QFormLayout" name="cameraAxisLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cameraAxisLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>cameraAxisX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="cameraAxisX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cameraAxisY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cameraAxisZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cameraAxisLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>cameraAxisY</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cameraAxisLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>cameraAxisZ</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="cameraPosGroup">
     <property name="title">
      <string>Camera position</string>
     </property>
     <layout class="QFormLayout" name="cameraPosLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cameraPosLabelX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="buddy">
         <cstring>cameraPosX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="cameraPosX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cameraPosLabelY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>cameraPosY</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cameraPosY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cameraPosLabelZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="buddy">
         <cstring>cameraPosZ</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cameraPosZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QGroupBox" name="infoGroup">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="contextLabel">
        <property name="text">
         <string>Context</string>
        </property>
        <property name="buddy">
         <cstring>context</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="context">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="identityLabel">
        <property name="text">
         <string>Identity</string>
        </property>
        <property name="buddy">
         <cstring>identity</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="identity">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
